dev/archery/archery/crossbow/reports.py

```diff
@@ -32,7 +33,6 @@ def __init__(self, job):
     def show(self):
         raise NotImplementedError()
 
-
```

Comment:
I suspect this new line was intentional, as archery uses black for formatting.

- https://github.com/apache/arrow/pull/10012
- https://github.com/psf/black

> Surround top-level function and class definitions with two blank lines.
- https://www.python.org/dev/peps/pep-0008/

Reply:
Nope, we didn't merge that eventually to better align with the formatting of pyarrow.